c/cjson/cjson.go

```diff
@@ -26,128 +26,645 @@ const (
+func (o *JSON) AddItemReferenceToObject(s *c.Char, item *JSON) Bool { return Bool(0) }
 
-// Render a JSON entity to text using a buffered strategy.
+// CJSON_PUBLIC(cJSON *) cJSON_DetachItemViaPointer(cJSON *parent, cJSON * const item);
 //
-// prebuffer is a guess at the final size. guessing well reduces reallocation.
+// Remove/Detach items from Arrays/Objects.
 //
-// fmt=0 gives unformatted, =1 gives formatted.
+// llgo:link (*JSON).DetachItemViaPointer C.cJSON_DetachItemViaPointer
+func (o *JSON) DetachItemViaPointer(item *JSON) *JSON { return nil }
+
+// CJSON_PUBLIC(cJSON *) cJSON_DetachItemFromArray(cJSON *array, int which);
 //
-// llgo:link (*JSON).PrintBuffered C.cJSON_PrintBuffered
-func (o *JSON) PrintBuffered(prebuffer c.Int, fmt c.Int) *c.Char { return nil }
+// llgo:link (*JSON).DetachItemFromArray C.cJSON_DetachItemFromArray
+func (o *JSON) DetachItemFromArray(which c.Int) *JSON { return nil }
 
-// llgo:link (*JSON).GetObjectItemCaseSensitive C.cJSON_GetObjectItemCaseSensitive
-func (o *JSON) GetObjectItemCaseSensitive(key *c.Char) *JSON { return nil }
+// CJSON_PUBLIC(void) cJSON_DeleteItemFromArray(cJSON *array, int which);
+//
+// llgo:link (*JSON).DeleteItemFromArray C.cJSON_DeleteItemFromArray
+func (o *JSON) DeleteItemFromArray(which c.Int) {}
 
-// llgo:link (*JSON).GetArraySize C.cJSON_GetArraySize
-func (o *JSON) GetArraySize() c.Int { return 0 }
+// CJSON_PUBLIC(cJSON *) cJSON_DetachItemFromObject(cJSON *object, const char *string);
+//
+// llgo:link (*JSON).DetachItemFromObject C.cJSON_DetachItemFromObject
+func (o *JSON) DetachItemFromObject(s *c.Char) *JSON { return nil }
 
-// llgo:link (*JSON).GetArrayItem C.cJSON_GetArrayItem
-func (o *JSON) GetArrayItem(index c.Int) *JSON { return nil }
+// CJSON_PUBLIC(cJSON *) cJSON_DetachItemFromObjectCaseSensitive(cJSON *object, const char *string);
+//
+// llgo:link (*JSON).DetachItemFromObjectCaseSensitive C.cJSON_DetachItemFromObjectCaseSensitive
+func (o *JSON) DetachItemFromObjectCaseSensitive(s *c.Char) *JSON { return nil }
 
-// llgo:link (*JSON).GetStringValue C.cJSON_GetStringValue
-func (o *JSON) GetStringValue() *c.Char { return nil }
+// CJSON_PUBLIC(void) cJSON_DeleteItemFromObject(cJSON *object, const char *string);
+//
+// llgo:link (*JSON).DeleteItemFromObject C.cJSON_DeleteItemFromObject
+func (o *JSON) DeleteItemFromObject(s *c.Char) {}
+
+// CJSON_PUBLIC(void) cJSON_DeleteItemFromObjectCaseSensitive(cJSON *object, const char *string);
+//
+// llgo:link (*JSON).DeleteItemFromObjectCaseSensitive C.cJSON_DeleteItemFromObjectCaseSensitive
+func (o *JSON) DeleteItemFromObjectCaseSensitive(s *c.Char) {}
+
+// CJSON_PUBLIC(cJSON_Bool) cJSON_InsertItemInArray(cJSON *array, int which, cJSON *newitem);
+//
+// Update array items.
+// Shifts pre-existing items to the right.
+//
+// llgo:link (*JSON).InsertItemInArray C.cJSON_InsertItemInArray
+func (o *JSON) InsertItemInArray(which c.Int, newitem *JSON) Bool { return Bool(0) }
```

Comment:
it is better to remove the original function signature once the mapping is complete, just keep the original comments